src/intro.adoc

```diff
@@ -221,7 +221,7 @@ integer variants, RV32I and RV64I, described in
 <<rv32>> and <<rv64>>, which provide 32-bit
 or 64-bit address spaces respectively. We use the term XLEN to refer to
 the width of an integer register in bits (either 32 or 64).
-<<rv32e, Chapter 6>> describes the RV32E and RV64E subset variants of the
+<<rv32e, Chapter 3>> describes the RV32E and RV64E subset variants of the
```

Comment:
Best to change from <<rv32e, Chapter 3>> to <<rv32e>> - the tool will insert the right chapter number.

Reply:
Done 👍🏽